modules/tensor_mechanics/include/auxkernels/CylindricalRankTwoAux.h

```diff
@@ -0,0 +1,43 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef CYLINDRICALRANKTWOAUX_H
+#define CYLINDRICALRANKTWOAUX_H
+
+#include "AuxKernel.h"
+#include "RankTwoTensor.h"
+
+class CylindricalRankTwoAux;
+
+/**
+ * CylindricalRankTwoAux is designed to take the data in the CylindricalRankTwoTensor material
+ * property, for example stress or strain, and output the value for the
+ * supplied indices in cylindrical coordinates, where the cylindrical plane axis is
+ * along the z-axis and the center point in the x-y plan is defined by by center_point.
+ */
+
+template<>
+InputParameters validParams<CylindricalRankTwoAux>();
+
+class CylindricalRankTwoAux : public AuxKernel
+{
+public:
+  CylindricalRankTwoAux(const InputParameters & parameters);
+  virtual ~CylindricalRankTwoAux() {}
+
+protected:
+  virtual Real computeValue();
+
+private:
+
+  const MaterialProperty<RankTwoTensor> & _tensor;
+  const unsigned int _i;
+  const unsigned int _j;
+  const Point _center_point;
+
```

Comment:
could you remove the empty lines after the `private:` and before the `}`. Also I don't see why we need those members to be private. I'd just remove that and let them be protected.